packages/vue/src/components/molecules/SfCard/SfCard.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="sf-card">
+    <!-- @slot Custom content that will replace default image-->
+    <slot name="image" v-bind="image">
+      <SfImage :src="image" class="sf-card__image" />
+    </slot>
+    <!-- @slot Custom content for heading and description-->
+    <slot name="details" v-bind="{ title, description, titleLevel }">
+      <div class="sf-card__details">
+        <SfHeading
+          :title-level="titleLevel"
+          :title="title"
+          class="sf-card__title"
+        />
+        <p class="sf-card__description">
+          {{ description }}
+        </p>
+      </div>
+    </slot>
+    <!-- @slot Slot content will replace button -->
+    <slot name="action">
+      <SfButton :link="link" class="sf-card__action" v-on="$listeners">
+        {{ buttonText }}
+      </SfButton>
+    </slot>
+  </div>
+</template>
+<script>
+import SfButton from "../../atoms/SfButton/SfButton.vue";
+import SfHeading from "../../atoms/SfHeading/SfHeading.vue";
+import SfImage from "../../atoms/SfImage/SfImage.vue";
+
+export default {
+  name: "SfCard",
+  components: {
+    SfButton,
+    SfHeading,
+    SfImage,
+  },
+  props: {
+    /**
+     * Card image
+     */
+    image: {
+      type: [String, Object],
+      default: "",
+    },
+    /**
+     * Card title
+     */
+    title: {
+      type: String,
+      default: "",
```

Comment:
this should be required.

Reply:
shouldn't we leave it to choose if anyone wants to have a title? We didn't use required anywhere, should we change our approach?

Reply:
A Card should have a title, shouldn't it? But you have a point, I think we can leave it as is.